spec/controllers/sufia/admin/admin_sets_controller_spec.rb

```diff
@@ -43,25 +43,27 @@
     end
 
     describe "#create" do
+      let(:service) { instance_double(Sufia::AdminSetService) }
+      before do
+        allow(Sufia::AdminSetService).to receive(:new)
```

Comment:
If you create `Sufia::AdminSetService.create` method you won't require this before preamble and can instead add the expectation directly in line 55. This helps with the Law of Demeter.

```ruby
expect(Sufia::AdminSetService).to receive(:create).with(AdminSet, user).and_return(true)
```